javaparser-symbol-solver-core/src/main/java/com/github/javaparser/symbolsolver/resolution/typesolvers/JavaParserTypeSolver.java

```diff
@@ -63,6 +69,11 @@ public JavaParserTypeSolver(File srcDir) {
             throw new IllegalStateException("SrcDir does not exist or is not a directory: " + srcDir.getAbsolutePath());
         }
         this.srcDir = srcDir;
+        javaParser = new JavaParser(
```

Comment:
Should this be possibly configurable? Maybe we could optionally pass a parser configuration

Reply:
I just opened an issue for that. I could just as well fix it right away :-)

Reply:
Fixed.